.travis.yml

```diff
@@ -1,9 +1,10 @@
 sudo: false
 language: node_js
 node_js: 6
+before_install:
+  - "sh -e /etc/init.d/xvfb start"
```

Comment:
The trick here, is instead of just polling xwininfo, xvfb can boot in parallel to an npm install.

Should speed the build up a little